src/main/java/org/jahia/commons/encryption/EncryptionUtils.java

```diff
@@ -136,7 +138,9 @@ public static String pbkdf2Digest(String source, boolean prefixWithId) {
      * @param source
      *            the source text to be digested
      * @return the Base64 encoded SHA-1 digest of the provided text
+     * @deprecated in Jahia 7 a more robust PBKDF2 algorithm is used for password hashing. The previous SHA-1 based algorithm is no longer used
      */
+    @Deprecated
```

Comment:
should we add a `since` parameter here?

Reply:
Not interesting in library that small I would say.